nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -155,9 +155,9 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
         try {
             session.exportTo(flowFile, response.getOutputStream());
             response.flushBuffer();
-        } catch (final IOException ioe) {
+        } catch (final Exception e) {
```

Comment:
Yes I agree with this approach. I though Exception was justified by the fact that if the exception is not caught, the request is not removed from the session and will be re-queued (and yield the processor) which can be blocking.
All things considered, it's cleaner to catch individual exceptions and I'll update the PR as suggested.